libs/application/templates/parental-leave/src/lib/ParentalLeaveTemplate.ts

```diff
@@ -626,6 +738,10 @@ const ParentalLeaveTemplate: ApplicationTemplate<
           },
           lifecycle: pruneAfterDays(970),
           progress: 1,
+          onExit: defineTemplateApi({
+            action: ApiModuleActions.validateApplication,
+            throwOnError: true,
+          }),
```

Comment:
What is the reason for calling validateApplication onExit?

Reply:
No need it is out